src/main/java/uk/gov/hmcts/reform/lrdapi/service/impl/RegionServiceImpl.java

```diff
@@ -18,13 +36,99 @@
     @Autowired
     RegionRepository regionRepository;
 
-    public LrdRegionResponse retrieveRegionByRegionDescription(String description) {
-        Region region = regionRepository.findByDescriptionIgnoreCase(description);
+    @Value("${loggingComponentName}")
+    private String loggingComponentName;
+
+    public Object retrieveRegionDetails(String regionId, String description) {
```

Comment:
If you are returning a `List<LrdRegionResponse>` in all the cases, then change the return type of this method to the same.

Reply:
Addressed here: https://github.com/hmcts/rd-location-ref-api/pull/273/commits/c0621fec5e909c80a9c5cd14e0133ff79eede9cd